ansible/medicina.yml

```diff
@@ -31,7 +30,7 @@
 
 
 - name: Install yum packages
-  hosts: all
+  hosts: manager
```

Comment:
This will cause that other machines like storage and console will not have yum packages installed